python/ray/tests/test_failure_2.py

```diff
@@ -31,7 +31,7 @@ def test_warning_for_too_many_actors(shutdown_only):
 
     p = init_error_pubsub()
 
-    @ray.remote
+    @ray.remote(num_cpus=0)
```

Comment:
it appears that this test needed to be changed, because with the fixed scheduling class, these actors are getting hit by worker capping now.